source/user-manual/registering/index.rst

```diff
@@ -8,56 +8,104 @@ Registering agents
-|                | :ref:`agent-verification-with-host-validation`                |
-|                +---------------------------------------------------------------+
-|                | :ref:`agent-verification-without-host-validation`             |
-+----------------+---------------------------------------------------------------+
+    .. code-block:: console
+
+     # /var/ossec/bin/agent-auth -m <manager_IP>
+
+   2. Edit the agent's  ``/var/ossec/etc/ossec.conf`` configuration file:
+
+    .. include:: ../../_templates/registrations/common/client_server_section.rst
+
+   3. Start the agent:
+
+    .. include:: ../../_templates/registrations/linux/start_agent.rst
+
+
+   The agent registration can be adjusted by using different :ref:`agent-auth` options.
+
+
+
+ .. group-tab:: Windows host
+
+   Open a Powershell or CMD session in the agent host as an ``Administrator``.
+
+    .. include:: ../../_templates/registrations/windows/installation_directory.rst
+
+   1. Run the ``agent-auth.exe`` program, using the manager's IP address:
+
+    .. include:: ../../_templates/registrations/common/set_agent_name.rst
+
+    .. code-block:: console
+
+     # C:\Program Files (x86)\ossec-agent\agent-auth.exe -m <manager_IP>
+
+   2. Edit the agent's ``C:\Program Files (x86)\ossec-agent\ossec.conf`` configuration file:
+
+    .. include:: ../../_templates/registrations/common/client_server_section.rst
+
+   3. Start the agent.
+
+    .. include:: ../../_templates/registrations/windows/start_agent.rst
+
+
+
+ .. group-tab:: MacOS X host
+
+  Open a session in the MacOS X agent host as a ``root`` user.
+
+  1. Run the ``agent-auth`` program, using the manager’s IP address:
+
+   .. include:: ../../_templates/registrations/common/set_agent_name.rst
+
+   .. code-block:: console
+
+    # /Library/Ossec/bin/agent-auth -m <manager_IP>
+
+  2. Edit the agent's ``/Library/Ossec/etc/ossec.conf`` configuration file:
+
+   .. include:: ../../_templates/registrations/common/client_server_section.rst
+
+  3. Start the agent:
+
+   .. include:: ../../_templates/registrations/macosx/start_agent.rst
+
+  The agent registration can be adjusted by using different :ref:`agent-auth` options.
+
+
+The above method is the simplest way of registering the agent. There are also available :ref:`other registration methods <other_registration_methods>`.
+
+To learn more about the agent registration process, please read the :ref:`registering agents - additional information <registering_agent_theory>`.
+
+In case of problems during the registration, several solutions can be found on :ref:`registering agents' troubleshooting <registering-agents-troubleshooting>`.
 
 .. toctree::
     :maxdepth: 2
     :hidden:
 
-    registration-process
-    using-command-line
-    simple-registration-method
-    password-authorization-registration-service
-    manager-verification/host-verification-registration
-    restful-api-register
+    other-registration-methods
```

Comment:
And I think that every other registration method must have a dedicated single page in order to avoid the user think that all the page steps must be completed in a registration alternative process.

Reply:
Done